lib/request.js

```diff
@@ -313,8 +313,13 @@ AWS.Request = inherit({
     var region = service.config.region;
     var customUserAgent = service.config.customUserAgent;
 
-    // global endpoints sign as us-east-1
-    if (service.isGlobalEndpoint) region = 'us-east-1';
+    if (service.isGlobalEndpoint) {
+      if (service.signatureRegion) {
+        region = service.signatureRegion;
```

Comment:
No need to fix in this PR but we need to expost `signingRegion` to the client config later.